views/home/doctorHome.php

```diff
@@ -27,11 +27,274 @@
+
+        $db_link = mysqli_connect("localhost", "root", "", "retire");
+
+        if ($db_link == false) {
+          die("ERROR: Could not connect. " . mysqli_connect_error());
+        }
+
+        // Check only one field can be filled
+        // then setting proper variables.
+        if (!empty($date)) {
+          $filter = $date . '%';
+          $column = $day;
+        }
+        else {
+
+          if (!empty($Fname)) {
+            $filter = $Fname . '%';
+            $column = $first;
+          }
+
+          else {
+
+              if (!empty($Lname)) {
+                $filter = $Lname . '%';
+                $column = $Last;
+              }
+
+              else {
+
+                if (!empty($Comment)) {
+
+                  $filter = $Comment . '%';
+                  $column = $com;
+                }
+
+                else {
+
+                  if (!empty($Mmed)) {
+
+                    $filter = $Mmed . '%';
+                    $column = $morn;
+
+                  }
+
+                  else {
+
+                    if (!empty($Amed)) {
+
+                      $filter = $Amed . '%';
+                      $column = $aft;
+
+                    }
+
+                    else {
+
+                      if (!empty($Nmed)) {
+
+                        $filter = $Nmed . '%';
+                        $column = $night;
+
+                      }
+
+                      else {
+
+                        echo '<p class="error">You have no fields filled out. Please fill in one field.</p>';
+
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        // TODO: Make sure to get only unique prescriptions may require adding day to
+        // prescriptions table.
+         $sql = "SELECT  a.day, p.morning_med, p.afternoon_med, p.night_med,
+                p.comment, u.Fname, u.Lname
+                FROM appointments as a JOIN prescriptions as p on (a.patient_id = p.patient_id)
+                JOIN users as u ON (a.patient_id=u.id)
+                WHERE doctor_id LIKE $id AND $column LIKE '$filter' AND day < '2020-11-19%'";
+
+         $result = mysqli_query($db_link, $sql);
+
+         if (empty($result)) {
+           die("<p class='error'>The field you entered has no results</p>");
+         }
+
+         else {
+
+           // If result is not empty then display table
+         echo '<table>';
+         echo '<tbody>';
+         echo '<tr>';
+         echo '<th>Date</th>';
+         echo '<th>First Name</th>';
+         echo '<th>Last Name</th>';
+         echo '<th>Comment</th>';
+         echo '<th>Morning Med</th>';
+         echo '<th>Afternoon Med</th>';
+         echo '<th>Night Med</th>';
+         echo '<th>Patient Page</th>';
+         echo '</tr>';
+
+         if ($result->fetch_assoc()) {
+
+            // Every result row displayed in table if there is any
+           while ($row = $result->fetch_assoc()) {
```

Comment:
I'm not confidant on this, but wouldn't running `fetch_assoc()` in the `if` cause the first record in row to be excluded?